packages/app/components/stackbox/Stackbox.tsx

```diff
@@ -91,11 +115,42 @@ export const Stackbox = () => {
         closeAction={closeTokenPicker}
         initialFocusRef={searchTokenBarRef}
         isOpen={isTokenPickerOpen}
+        onTokenSelect={isPickingTokenFrom ? setTokenFrom : setTokenTo}
       />
       <ConfirmStackModal
         isOpen={isConfirmStackOpen}
         closeAction={closeConfirmStack}
       />
+    </>
+  );
+};
+
+const SelectTokenChip = ({ label, onClick, token }: SelectTokenChipProps) => {
+  const isTokenFrom = label === TOKEN_FROM_LABEL;
```

Comment:
I think I would like something more like this:

```suggestion
  const isTokenFrom = `label.toLowerCase().includes("deposit")`;
```

And then we could remove the constants: `TOKEN_FROM_LABEL` and `TOKEN_TO_LABEL`